src/frontends/sql/decoder_context.h

```diff
@@ -36,30 +36,32 @@ class DecoderContext {
   static constexpr absl::string_view kDefaultOutputName = "out";
 
   DecoderContext(ir::IRContext &ir_context)
-    : ir_context_(ir_context),
-      literal_operator_(ir_context_.RegisterOperator(
-          std::make_unique<ir::Operator>(kSqlLiteralOpName))) { }
+      : ir_context_(ir_context),
+        literal_operator_(ir_context_.RegisterOperator(
+            std::make_unique<ir::Operator>(kSqlLiteralOpName))),
+        global_module_(),
+        top_level_block_builder_() {}
 
   // Register a value as associated with a particular id. This stores that
   // value and provides a reference to its canonical value.
   const ir::Value &RegisterValue(uint64_t id, ir::Value value) {
     auto insert_result =
         id_to_value.insert({id, std::make_unique<ir::Value>(std::move(value))});
     CHECK(insert_result.second)
-      << "id_to_value map has more than one value associated with the id "
-      << insert_result.first->first << ".";
+        << "id_to_value map has more than one value associated with the id "
```

Comment:
nit: why do we have these formatting changes now? Have  you started using `clang-format` recently?

Reply:
I have. I didn't have it set up properly before, and now I'm frequently using it to keep my formatting standard.